src/lib/Components/Consignments/Screens/Metadata.tsx

```diff
@@ -219,9 +219,17 @@ export default class Metadata extends React.Component<Props, State> {
                     }}
                     style={{ margin: 10 }}
                   />
-                  <View style={{ flex: 1, flexDirection: "row", alignItems: "center", margin: 10 }}>
-                    <Label>Units</Label>
-                    <Toggle selected={this.state.unit === "CM"} left="CM" right="IN" onPress={this.updateUnit} />
+                  <View
+                    style={{
+                      flex: 1,
+                      flexDirection: "row",
+                      alignItems: "center",
+                      margin: 10,
+                      justifyContent: "space-between",
+                    }}
+                  >
+                    <Serif size="4">Units</Serif>
+                    <Toggle selected={this.state.unit === "CM"} left="Cm" right="In" onPress={this.updateUnit} />
```

Comment:
[The first result on Google](http://englishplus.com/grammar/00000058.htm) says that these should be lowercase. 

```suggestion
                    <Toggle selected={this.state.unit === "CM"} left="cm" right="in" onPress={this.updateUnit} />
```

I only knew to check because, as a lifelong user of the metric system `"Cm"` looks _really_ weird to me.